Bio/PDB/Residue.py

```diff
@@ -54,19 +96,8 @@ def sort(self):
         by name, with any alternative location specifier for disordered
         atoms (altloc) as a tie-breaker.
         """
-        # Defining sort key function within the sort method's scope:
-        def sort_index(atom):
-            """Build tuple of (int, name, alt-loc) for sorting.
-
-            The first integer is 0, 1, 2, 3, 4 for atoms N, CA, C, O, other.
-            """
-            try:
-                i = ["N", "CA", "C", "O"].index(atom.name)
-            except ValueError:
-                i = 4
-            return (i, atom.name, atom.altloc)
-
-        self.child_list.sort(key=sort_index)
+
+        self.child_list.sort
```

Comment:
Should that not be ``self.child_list.sort()`` for an in-place sorting of the list?